src/sql/media/icons/common-icons.css

```diff
@@ -528,6 +528,20 @@ Includes non-masked style declarations. */
 	background-image: url("database_colored.svg");
 }
 
+.notebookExplorer {
+	background-color: rgba(255, 255, 255, 0.4);
+	-webkit-mask-image: url("book_image.svg");
+	-webkit-mask-repeat: no-repeat;
+	-webkit-mask-position: 50% 50%;
+}
+
+.checked > .notebookExplorer {
```

Comment:
What is this checked class coming from? This feels like it should be somewhere else - it doesn't seem like this would be more scoped down